### DISCUSSION TITLE
Is it required to reference HandyControl and HandyControls nuget packages both ?

### DISCUSSION THREAD

**Discussion by YBAZAN** [HandyOrg/HandyControl#757] [Q&A] [answered]
In the quick start, it is not clear if we have to reference HandyControl and HandyControls nuget packages both or if HandyControls is enough because it contains the other one ?

Thank for your help

**Comment by ghost1372** [comment, marked as answer]
@YBAZAN  You have to choose one of these, **HandyControl** is the official version, and **HandyControls** is the unofficial version (Which has all the features of the official version + more features) So you do not need to refer to both, you just have to choose one of them
